src/pykiso/test_coordinator/test_execution.py

```diff
@@ -48,6 +49,7 @@ class ExitCode(enum.IntEnum):
     ONE_OR_MORE_TESTS_FAILED = 1
     ONE_OR_MORE_TESTS_RAISED_UNEXPECTED_EXCEPTION = 2
     ONE_OR_MORE_TESTS_FAILED_AND_RAISED_UNEXPECTED_EXCEPTION = 3
+    FAILED_AT_AUXILIARY_CREATION = 4
```

Comment:
```suggestion
    AUXILIARY_CREATION_FAILED = 4
```
Just to keep a similar format than the other members

Reply:
ohh there is a format ???